data/gegno/gegno intro missions.txt

```diff
@@ -432,6 +432,312 @@ mission "Giaru Gegno: Revisit"
 				decline
 
 
+mission "Suspicious Scin"
+	invisible
+	landing
+	source "Giaru Gegno"
+	destination "Yiia Iyr"
+	to offer
+		has "Visit Quarg in Gegno Space: offered"
+		not "First Contact: Gegno Scin: offered"
+		not "Gegno Genocide Defense: offered"
+	on enter
+		system
+			near "Aleit" 4 100
+		conversation
+			action
+				set "gegno: mysterious device"
+			`As you're traveling through Gegno space, a faint signal starts emanating from somewhere in your ship. You aren't sure what it is, as it doesn't appear to originate from the ship itself, so you decide to take a look. After a few minutes of searching for the culprit, you come across a device not too dissimilar from the one that the Gegno you met on Var Oti had given you, arbitrarily shoved into some corner of your ship's infrastructure. You don't know how or when it got there, but at least it wasn't something more dangerous like an explosive.`
```

Comment:
I don't really think that to be the case. The player wouldn't know what a strange signal in the ship is if they've never seen it before - it's not a matter of thinking about it, it's a fact that they don't know what it is. Telling them what they think would be more something like, "You don't like it because you don't know it", or, "You are interested in it", etc..